src/language-css/clean.js

```diff
@@ -61,9 +61,13 @@ function clean(ast, newObj) {
     newObj.importPath = cleanCSSStrings(newObj.importPath);
   }
 
-  if (ast.type === "selector-attribute" && newObj.value) {
-    newObj.value = newObj.value.replace(/^['"]|['"]$/g, "");
-    delete newObj.quoted;
+  if (ast.type === "selector-attribute") {
+    newObj.attribute = newObj.attribute.replace(/ /g, "");
```

Comment:
Can't there be other types of whitespace we want to give the same treatment?

Reply:
@lydell hm, maybe, let's test and fix this :+1: